src/traces/pie/index.js

```diff
@@ -373,6 +373,12 @@ pie.plot = function(gd, cdpie) {
                     }
                 }
 
+                function handleClick (evt) {
+                    gd._hoverdata = pt;
```

Comment:
I'd vote for returning an array of length 1 for consistency, even though as you said there will always be only one pt in that array.

cc @alexcjohnson thoughts?